doc/cug.tex

```diff
@@ -1236,35 +1236,31 @@ \subsection{Locating Suite And Task Output}
 \lstset{language=transcript}
 \begin{lstlisting}
 shell$ tree $HOME/cylc-run/tut.oneoff.basic/
-|-- cylc-suite.db      # suite run database
-|-- cylc-suite-env     # suite environment file
-|-- share              # suite share directory (not used in this example) 
-|-- work               # task work space (sub-dirs are deleted if not used)
-|    `-- 1                # task cycle point directory (or 1)
-|        `-- hello           # task work directory (deleted if not used)
-|-- log                # suite log directory
-|   `-- suite             # suite daemon log directory
-|   |   |-- err              # suite daemon stderr log (daemon mode only)
-|   |   |-- out              # suite daemon stdout log (damon mode only)
-|   |   `-- log              # suite daemon event log (timestamped info)     
-|   `-- job               # task job log directory
-|       `-- 1                # task cycle point directory (or 1)
-|           `-- hello           # task name
-|               |-- 01             # task submission number
-|               |   |-- job           # task job script
+|-- cylc-suite-private.db # private suite run database
```

Comment:
It's sort of a natural fit - `rose suite-run` (and maybe `cylc suite-run`) tar up old log directories without any database information, making it hard to view past runs in a web viewer like `Rose Bush`. It's probably not desirable for large suites, but for most it would be a good thing. Of course, we could just `mv` the database file ourselves on suite-run.